.circleci/config.yml

```diff
@@ -841,29 +841,29 @@ workflows:
           requires: [ 'Code Checkout' ]
           name: "PHP 70 Web integration tests"
           integration_testsuite: "test-web-70"
-          docker_image: "datadog/docker-library:ddtrace_alpine_php-7.0-debug"
-          lib_curl_command: sudo apk update ; sudo apk add curl-dev
+          docker_image: "datadog/dd-trace-ci:php-dev-7.0"
```

Comment:
Just making a documentation note for the future here: these images switch the Linux flavor from Alpine to Debian.